plugin/trino-iceberg/pom.xml

```diff
@@ -15,7 +15,7 @@
 
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
-        <dep.iceberg.version>0.13.2</dep.iceberg.version>
+        <dep.iceberg.version>0.14.0</dep.iceberg.version>
```

Comment:
Do you want to contribute to `trinodb/docker-images` as well? 
https://github.com/trinodb/docker-images/blob/master/testing/spark3.0-iceberg/Dockerfile#L17

Reply:
Thanks for pointing this out @ebyhr, I'll open a PR

Reply:
@ebyhr docker image update is here [#127](https://github.com/trinodb/docker-images/pull/137)